cms_form_compassion/forms/match_partner_form.py

```diff
@@ -8,183 +8,182 @@
 ##############################################################################
 import re
 
-from odoo import api, models, fields, tools, _
-
-testing = tools.config.get("test_enable")
+from odoo import api, models, fields, _
 
 
 @api.model
 def _lang_get(self):
     return self.env["res.lang"].sudo().get_installed()
 
 
-if not testing:
-    # prevent these forms to be registered when running tests
-
-    class PartnerMatchform(models.AbstractModel):
-        """A form that can create partner given data.
-           It will put the found partner in field partner_id that must
-           be present on the related model.
+class PartnerMatchform(models.AbstractModel):
+    """A form that can create partner given data.
+       It will put the found partner in field partner_id that must
+       be present on the related model.
+    """
+
+    _name = "cms.form.match.partner"
+    _description = "Form with partner matching"
+
+    _inherit = ["cms.form", "res.partner.match"]
+
+    partner_id = fields.Many2one("res.partner", readonly=False)
+    partner_title = fields.Many2one(
+        "res.partner.title", "Title", required=True, readonly=False
+    )
+    partner_firstname = fields.Char("First Name", required=True)
+    partner_lastname = fields.Char("Last Name", required=True)
+    partner_email = fields.Char("Email", required=True)
+    partner_phone = fields.Char("Phone", required=True)
+    partner_street = fields.Char("Street", required=True)
+    partner_zip = fields.Char("Zip", required=True)
+    partner_city = fields.Char("City", required=True)
+    partner_country_id = fields.Many2one(
+        "res.country", "Country", required=True, readonly=False
+    )
+    partner_state_id = fields.Many2one("res.country.state", "State", readonly=False)
+    partner_lang = fields.Selection(_lang_get, "Language")
```

Comment:
should be "_lang_get"

Reply:
I used lambda instead. The string would be wrong, the parameter should be a list of tuples.